testing/ponysay/APKBUILD

```diff
@@ -0,0 +1,21 @@
+# Maintainer: Matthew T Hoare <matthew.t.hoare@gmail.com>
+# Contributor: Sven-Hendrik Haase <sh@lutzhaase.com>
+# Contributor: Erkin Batu Altunbaş <erkinbatu@gmail.com>
+
+pkgname=ponysay
+pkgver=3.0.2
+pkgrel=0
+pkgdesc="cowsay reimplemention for ponies"
+url="http://erkin.github.com/ponysay/"
+arch="noarch"
+license="GPL3"
+depends="python bash"
+subpackages="$pkgname-doc"
+source="ponysay-$pkgver.tar.gz::https://github.com/erkin/ponysay/tarball/$pkgver"
```

Comment:
I believe it's recommended to use the actual releases instead of directly getting tarballs from tags. If you use "https://github.com/erkin/ponysay/archive/$pkgver.tar.gz" as the source, the output dir will be predictable too (project-ver, instead of name-project-hash).

That might also fix the issue with the hash not matching.